LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 <author>
+Copyright (c) 2024 Mister Fillname
```

Comment:
Miss Fillname ?

Reply:
How about Mx. Fillname ?

Reply:
But I think <author> was also okay.

Reply:
With \<author\> it might still have worked. But for example with \<email\> the tests failed because \<email\> did not have the proper form.

The only reason why I did the change was to give everything a form as it would occur in a real project to be able to test the project. It was not even intended as a joke. I just wanted to have Fillname in here to have unique token to replace. We can also use `Fillname Author` or similar if you think that is better.

Reply:
A, i just confused it with `fillname` with a small `f`.
If I understood your `init.py` file correctly, the  words that get replaced are:

```
author@fillname.com
Mister Fillname
https://fillname.org/
fillname
```

Is this correct ? Maybe we can unify this a little, as `fillname` is included in the email replacement itself, which can again be confusing. (Sorry that I get so easily confused)

Isn`t easier to use something more unique, like
```
<email>
<author>
<url>
<projectname>
```

Or did I get it all wrong?

Reply:
> Ah, and now I also understand your comment as you wanted the template to be fit the regular expressions. But I still don`t get why this could be useful. To test the project-template one would simply choose 4 fake things and then test for them?

If we use these values, the test workflow will fail because the project template cannot be installed (which is required for testing). The test workflow also runs for the template.

Reply:
We could make the values a bit more consistent:
```
author@fillname.org
Author Fillname
https://fillname.org/
fillname
```

Reply:
OK, I tried to replace it the way I suggested in my own project and could see the error you meant. Basically the pyproject.toml isnt able to install a project called "<project>".
https://github.com/MaxOstrowski/test-template/

Not sure this is a bad thing though. It prevents the user from using the project without executing `init.py` first.
On the other hand it's not looking too nice if the CI test in the template repo fail.
Maybe your more consistent set of names is a good tradeoff.

Reply:
> Maybe your more consistent set of names is a good tradeoff.

I think it is. Having the ci tests running, will ensure that we do not have unnecessary errors in the python code as we continue extending the template.

We could also extend the CI tests to run the init script but that would be a bit cumbersome to implement.

The current approach is an easy workaround with somewhat curios placeholders that is otherwise nice and simple.